client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Flex, Heading, Text, Link } from '@chakra-ui/layout';
+import { useConfirmDelete } from 'chakra-confirm';
+import { useRouter } from 'next/router';
+import { Button } from '@chakra-ui/button';
+import { useDeleteMeMutation } from 'generated/graphql';
+import { useAuth } from 'modules/auth/store';
```

Comment:
```suggestion
import { useAuthStore } from 'modules/auth/store';
```
By using the store we can update the stored user after deleteMe resolves.